wire/internal/wire/wire_test.go

```diff
@@ -309,9 +320,15 @@ func loadTestCase(root string, wireGoSrc []byte) (*testCase, error) {
 	if err != nil {
 		return nil, fmt.Errorf("load test case %s: %v", name, err)
 	}
+	var wireGold []byte
 	wantErrorStrings, wantError := parseGoldenOutput(out)
```

Comment:
Suggest renamiong "parseGoldenOutput" to "parseWantOutput" or something like that, since "golden" means something else now.